pages/Advanced Queries.md

```diff
@@ -1,4 +1,4 @@
-- The database that Logseq used is [[https://github.com/tonsky/datascript][Datascript]], which is an immutable in-memory database and [Datalog](https://en.wikipedia.org/wiki/Datalog) query engine in Clojure and ClojureScript.
+- The database that Logseq used is [Datascript](https://github.com/tonsky/datascript), which is an immutable in-memory database and [Datalog](https://en.wikipedia.org/wiki/Datalog) query engine in Clojure and ClojureScript.
```

Comment:
`used` should be replaced with `use`. 

- ~~The database that Logseq used~~
+ The database that Logseq use